sphinx/texinputs/Makefile_t

```diff
@@ -15,38 +21,33 @@ FMT = pdf
 
 {% if latex_engine == 'platex' -%}
 # latexmkrc is read then overridden by latexmkjarc
-LATEX = latexmk -r latexmkjarc -pdfdvi -dvi- -ps-
-PDFLATEX = $(LATEX)
-
-all-dvi: $(ALLIMGS) $(ALLDVI)
-
-all-ps: $(ALLPS)
-
-all: all-dvi $(ALLPDF)
-
-all-pdf: all-dvi $(ALLPDF)
-{% else -%}
-# latexmkrc contains config for pdflatex, lualatex, xelatex
+LATEX = latexmk -r latexmkjarc -dvi
+PDFLATEX = latexmk -r latexmkjarc -pdfdvi -dvi- -ps-
+{% elif latex_engine == 'pdflatex' -%}
 LATEX = latexmk -dvi
-{% if latex_engine == 'pdflatex' -%}
-LATEXMKPDFOPT = -pdf
+PDFLATEX = latexmk -pdf -dvi- -ps-
 {% elif latex_engine == 'lualatex' -%}
-LATEXMKPDFOPT = -pdflua
 LATEX = latexmk -lualatex
+PDFLATEX = latexmk -pdflua -dvi- -ps-
 {% elif latex_engine == 'xelatex' -%}
-LATEXMKPDFOPT = -pdfxe
-{% else -%}
-{% endif -%}
-PDFLATEX = latexmk $(LATEXMKPDFOPT) -dvi- -ps-
-
+# latexmk  $xelatex variable configured in latexmkrc to generate xdv files only
+LATEX = latexmk -xelatex
+PDFLATEX = latexmk -pdfxe -dvi- -ps-
+{% endif %}
+{% if latex_engine != 'xelatex' %}
 all-dvi: $(ALLIMGS) $(ALLDVI)
 
 all-ps: $(ALLPS)
+{% endif -%}
+{% if latex_engine == 'platex' %}
+all: all-dvi $(ALLPDF)
 
+all-pdf: all-dvi $(ALLPDF)
+{% else %}
 all: $(ALLPDF)
 
 all-pdf: $(ALLPDF)
-{% endif %}
+{%- endif %}
 
 %.png %.gif %.jpg %.jpeg:
 	extractbb $@
```

Comment:
Nice catch.. 
I don't know whether `extractbb *.pdf` is needed or not.
In fact, In my environment was crushed if `%.pdf` is specified.